--- /usr/sbin/fai-make-nfsroot.orig	2015-05-26 06:12:08.000000000 -0500
+++ /usr/sbin/fai-make-nfsroot	2015-11-24 12:59:41.000000000 -0500
@@ -69,7 +69,7 @@
     fi
 
     echo "FAI packages and related packages inside the nfsroot:"
-    $ROOTCMD dpkg-query -W -f='${Package;-18} ${Version}\n' fai-client fai-nfsroot fai-setup-storage dracut-network dracut dracut-live dracut-squash 2>/dev/null
+    $ROOTCMD aptitude -Rfy install fai-nfsroot=5.10.3 fai-setup-storage=5.10.3 fai-client=5.10.3 dracut-network dracut dracut-live dracut-squash
 
     # check if all important packages are installed
     n=$($ROOTCMD dpkg-query -l fai-client fai-nfsroot fai-setup-storage dracut-network dracut 2>/dev/null|grep -c -E ^ii)
